tests/baselines/reference/aliasUsageInTypeArgumentOfExtendsClause.types

```diff
@@ -25,7 +25,7 @@ class C<T extends IHasVisualizationModel> {
 }
 class D extends C<IHasVisualizationModel> {
 >D : D
->C : C<T>
+>C : typeof C

```

Comment:
The best way to handle all these changes is to tweak typeWriter to produce results like what we used to get.  Then, we can actually review the handful of files that have relevant changes.  Then, once we can see that nothing else changed unintentionally, we can remove that tweak in a followup change and then update all the baselines.  With several hundred files changed, github won't show all the changes.  And it becomes very difficult to ascertain if something else might have broke as part of this.

We've done this numerous times as we've changed hte compiler.  For example, even when moving over from the old compiler to the new one, we tweaked things in the new typewriter to produce the same output as hte previous one.